code/datums/trading/unique.dm

```diff
@@ -25,6 +25,64 @@
 /datum/trader/ship/unique/what_do_you_want()
 	return get_response("what_want", "I don't want anything!")
 
+/datum/trader/ship/unique/syndicate
+	name = "Cyndie Kate"
+	origin = "Cloaked ship"
+
+	possible_wanted_items = list (
+		/obj/item/weapon/gun/energy/captain					= TRADER_THIS_TYPE,
+		/obj/item/weapon/hand_tele							= TRADER_THIS_TYPE,
+		/obj/item/blueprints								= TRADER_THIS_TYPE,
+		/obj/item/weapon/disk/nuclear						= TRADER_THIS_TYPE,
+		/obj/item/weapon/reagent_containers/hypospray		= TRADER_ALL,
+		/obj/item/weapon/storage/box/ids					= TRADER_THIS_TYPE,
```

Comment:
That´s a bad item, because this is just the box with a label of "spare id box", which does not necessarily need to contain any ids

Reply:
The id box has a specific graphic and is distinct from other boxes in the code as well.
![image](https://user-images.githubusercontent.com/6859396/62106106-a25e9900-b2a4-11e9-80d2-dcf4556423f3.png)

However, I agree we could change it to trading for ID's only - this would allow collection of ID's from slain security guards as dog tags to be sold to the syndicate merchant. Your call.

Reply:
As I said. Its a definite removal.
Why would someone pay you something for a cardboard box with a painting on it.

Reply:
Makes sense, will remove.